### PR TITLE
Basic documentation

### PR DESCRIPTION
**Pull request by mukobi**
Create a basic README.md (adopted from https://gist.github.com/PurpleBooth/109311bb0361f32d87a2) and CONTRIBUTING.md (adopted from Knative GitHub repos) for the master project.

### CODE REVIEW

**Comment by mukobi** [comment]
Whoops, I accidentally made this branch from knative-portability/large-events and not my own fork. The Cloud Build trigger is failing because I set up that trigger to build the posts subdirectory but the PR bringing in posts hasn't been merged yet.

File: README.md [hunk 1]
```diff
@@ -1,3 +1,101 @@
...
+## Introduction
+
+An app built as proof of portability for [Knative](https://knative.dev) that provides a platform for event organizers to share info during large events like concerts, parades, expos, etc. Key features of this app include:
+* Allow authorized event organizers to create and edit events
+* Handle uploading and serving multimedia posts made by attendees
+* Organize and filter multiple sub-events
+* Provide a clean front-end web UI for interaction with the service
+* Authenticate with OAuth 2.0
+
+This app is built as part of a proof of portability project for [Knative](https://knative.dev). It is meant to show the key features of Knative, to test the conformance various cloud product implementations of Knative like [Google Cloud Run](https://cloud.google.com/run/), and to document with functioning sample code how one might develop, build, and deploy with Knative. Key features of Knative this app demonstrates include:
```
Comment (evankanderson) [inline README.md]: I'd remove the Cloud Run reference here, and instead have a section on "where to run this?" that references that it's been tested on Cloud Run.

Comment (evankanderson) [reply README.md]: (And then we can add to the list as we test it other places.)

File: README.md [hunk 2]
```diff
@@ -1,3 +1,101 @@
...
+Set up and activate a virtual environment.
+
+```
+python3 -m venv venv && . venv/bin/activate
+```
+
+Install the required python modules.
+
+```
+pip3 install requirements.txt
```
Comment (evankanderson) [inline README.md]: install **-r** requirements.txt ?

File: README.md [hunk 3]
```diff
@@ -1,3 +1,101 @@
...
+
+```
+gcloud builds submit --tag gcr.io/[PROJECT-ID]/[SERVICE]
+gcloud beta run deploy --image gcr.io/[PROJECT-ID]/[SERVICE] --platform managed
+```
+
+## Built With
+
+* [Flask](http://flask.pocoo.org/) - Python web service framework
+* [MongoDB Atlas](https://www.mongodb.com/cloud/atlas) - Cloud NoSQL database
```
Comment (evankanderson) [inline README.md]: If MongoDB needs to be provisioned, you might want to mention it in the "Deployment" section.

File: README.md (hunk 1 above)
Comment (evankanderson) [reply README.md]: The "Deployment" section seems to cover this nicely.

Comment (mukobi) [reply README.md]: Done https://github.com/knative-portability/large-events/pull/6/commits/5a3aea8b819ebec42eb9f5e5dfa6b57c68588a4c

File: README.md (hunk 2 above)
Comment (mukobi) [reply README.md]: Done https://github.com/knative-portability/large-events/pull/6/commits/d43b1a3a7ca0eb76e8ab5f58c3578202af25cbd4

Comment (mukobi) [reply README.md]: Thanks for the catch

File: README.md (hunk 3 above)
Comment (mukobi) [reply README.md]: Done https://github.com/knative-portability/large-events/pull/6/commits/1234525d514f077d41dae372ca64598e2beb2c0e

File: README.md [hunk 4]
```diff
@@ -1,4 +1,110 @@
...
+## Introduction
+
+An app built as proof of portability for [Knative](https://knative.dev) that provides a platform for event organizers to share info during large events like concerts, parades, expos, etc. Key features of this app include:
+* Allow authorized event organizers to create and edit events
+* Handle uploading and serving multimedia posts made by attendees
+* Organize and filter multiple sub-events
+* Provide a clean front-end web UI for interaction with the service
+* Authenticate with OAuth 2.0
+
+This app is built as part of a proof of portability project for [Knative](https://knative.dev). It is meant to show the key features of Knative, to test the conformance various cloud product implementations of Knative, and to document with functioning sample code how one might develop, build, and deploy with Knative. Key features of Knative this app demonstrates include:
```
Comment (cmei4444) [inline README.md]: > "to test the conformance various cloud product implementations of Knative"
slight typo but could be confusing to readers - should it be 'conformance across various ..." or something similar?

Comment (mukobi) [reply README.md]: Thank you, fixed in https://github.com/knative-portability/large-events/pull/7

**Pull request merged**